fake_file.md

```diff
@@ -1,3 +1,6 @@
 Hey, _here's_ some neat content!
 
 Plus, I can just add lines.
+
+
+How far should I do?
```

Comment:
Making the above, I had the option to "Add single comment" or "Start a review" (chose the former)